spec/plugin_spec.rb

```diff
@@ -20,28 +20,26 @@
     subject
   end
 
-  def api_post(url, params = {})
+  def api_post(url, params = {}, expect_status: 200)
```

Comment:
I don't know exactly why but to my eyes mixing parameters with default values and keyword arguments looks like a bug.